docs/src/js/components/ElementsRadio.react.js

```diff
@@ -168,7 +168,7 @@ export default class RadioGroupPillSample extends React.Component {
                     <Radio.Item label="Option 03" />
                 </Radio><br /><br />
 
-                <Radio checked={this.state.checkedPill} name="pill-onchange-sample" onChange={this._onPillChange.bind(this)} pill={true}>
+                <Radio checked={this.state.checkedPill} name="pill-onchange-sample" onChange={this._onPillChange.bind(this)} pill={true} multi={true}>
```

Comment:
Small nitpick (no need to fix this right now), let's line break these attributes.

Reply:
Another nitpick, when defining a boolean attribute and its value is true, let's omit the value.